pa11y.json

```
@@ -0,0 +1,10 @@
+{
+  "defaults": {
+    "concurrency": 4,
+    "standard": "WCAG2AA",
+    "runners": ["axe"]
+  },
+  "chromeLaunchConfig": {
+      "args": ["--no-sandbox"]
+    }
+}
```

https://github.com/pa11y/pa11y-ci?tab=readme-ov-file#docker


```suggestion
{
    "defaults": {
        "chromeLaunchConfig": {
            "args": [
                "--no-sandbox"
            ]
        }
    }
}
```

---

Thanks for that. I'll try it.